### DISCUSSION TITLE
pyzeebe LongTime Running Task

### DISCUSSION THREAD

**Discussion by seb-835** [camunda-community-hub/pyzeebe#238] [Q&A]
Hi,
my worker is computing  long time task , to avoid it to pop a new task for the job, because the first one is not finished, i set the task timeout to a big_value. 

Is it possible to use value 0 or -1 for timeout  to disable the re_activation ? 

Best regards

**Comment by JonatanMartens** [comment]
Hey @seb-835. Unfortunately, Zeebe itself doesn't allow this behavior. 

See https://docs.camunda.io/docs/reference/grpc/

> GRPC_STATUS_INVALID_ARGUMENT
>>  timeout less than 1 (ms)

Maybe message events can solve this problem? See https://docs.camunda.io/docs/reference/bpmn-processes/message-events/message-events/

**Comment by seb-835** [comment]
Thanks @JonatanMartens , yes message events is the right solution.
